### PR TITLE
Typo and completiness

### PR DESCRIPTION
**Pull request by PhilippePerret**
Readme file modifications

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -360,6 +360,7 @@ Available letter casing settings are:
 Available whitespace settings are:
 ```ruby
 :trim     # remove whitespace from both ends of the input
+:strip    # idem
```
Comment (piotrmurach) [inline README.md]: What is `idem`? Could we expand the param behaviour to a simple sentence, please?

Comment (PhilippePerret) [reply README.md]: Sorry, 'idem' "the same as above".
We could use `# remove whitespace from both ends of the input` (as above)

Comment (piotrmurach) [reply README.md]: Can we say then 'same as :trim' or similar? I prefer plain English to acronyms that not many people may understand including myself 😄

Comment (PhilippePerret) [reply README.md]: Sorry, I'm french and 'idem' is a current word (not a "acronym").
'same as :trim' sounds perfect! :-D

**Comment by piotrmurach** [comment]
Thanks ❤️

**Pull request merged**